plugins/tailor.cpp

```diff
@@ -698,6 +712,7 @@ static void tailor_setMaterialPreferences(color_ostream &out, int32_t silkIdx,
     set_config_val(config, CONFIG_CLOTH_IDX, clothIdx - 1);
     set_config_val(config, CONFIG_YARN_IDX, yarnIdx - 1);
     set_config_val(config, CONFIG_LEATHER_IDX, leatherIdx - 1);
+    set_config_val(config, CONFIG_ADAMANTINE_IDX, leatherIdx - 1);
```

Comment:
```suggestion
    set_config_val(config, CONFIG_ADAMANTINE_IDX, adamantineIdx - 1);
```

Reply:
function needs a new `int32_t adamantineIdx` param.

Reply:
this is what i get for coding while still on my first cup of coffee